cmd/olfaucet/main.go

```diff
@@ -259,12 +259,12 @@ func (f *Faucet) RequestOLT(req Request, reply *Reply) error {
 	rawTx := sendTxResults.RawTx
 	h, err := f.nodeCtx.PrivKey().GetHandler()
 	if err != nil {
-		panic("invalid nodeCtx private key")
+		return errors.New("invalid nodeCtx private key")
```

Comment:
this will make the faucet running with errors, we should check the key is correct when launch the faucet.

Reply:
what is the suggestion for this? Should I just cancel the pull request and leave it as a panic instead?